src/backends/tensorflow/graphdef_backend_factory.cc

```diff
@@ -69,10 +69,18 @@ GraphDefBackendFactory::CreateBackend(
         std::make_tuple(graphdef_path));
   }
 
+  // Initialize virtual device counter
+  for (size_t gpu_idx = 0; gpu_idx < backend_config_->memory_limit_mb.size();
+       gpu_idx++) {
+    virtual_device_ids_.emplace(
```

Comment:
We need a map from physical GPU -> virtual GPU. This seems to be a map from virtual GPU -> ?.

Reply:
This does map [0, #gpus) ->  [0, #vgpus). The memory_limit_mb vector has #gpus elements in it.